README.md

```diff
@@ -60,7 +62,9 @@ $ docker-compose up -d app
 ## Running the tests
 
 ```sh
-# On the initial run or if the schema changes
+# On the initial run or if the schema changes.
+# The db container must already be running, or the tests will probably
+# fail on the first run since the database needs time to warm up.
```

Comment:
Having the explanation in these comments as 2 sentences (one that isn't really a sentence) reads a little awkwardly to me. I would probably move the explanatory text outside of comments, but if you prefer to leave them here, I am ok with that.

Reply:
done....